lib/index.js

```diff
@@ -944,6 +949,19 @@ Stream.prototype._removeConsumer = function (s) {
 };
 
 /**
+ * Removes an observer from this Stream.
+ */
+
+Stream.prototype._removeObserver = function (s) {
+    this._observers = this._observers.filter(function (o) {
+      return o !== s;
```

Comment:
This line and 960 are indented two spaces instead of four.

Side note: eslint was OK with this? Looks like we need to add a new rule...